### ISSUE TITLE
Where do i change the anchors ? and tips for improving fps on certain models?

### ISSUE THREAD

**Issue by Mahaaveer** [ardianumam/Tensorflow-TensorRT#20]
Hi,

I have a darknet weight for three classes trained with recalculated anchors.
I followed instructions from https://github.com/AlexeyAB/darknet  
When i try this weights on your TensorRT implementation, the bounding boxes are off and are large in size.
Where do i update with my new anchor values? i searched through utils.py and i can only find anchor masks.

Regarding performance........

On pretrained yolo v3, it gives 30 fps
On two classes yolo v3 model, it gives 20 fps
On three classes yolo v3 model ( the one with anchors problem), it gives 35fps !!

I am trying to deploy the two classes yolo model which is fast enough to serve on real life applications. i'd like to have 25+ fps
When i add a third class into the mix, it gave me 35 + fps !
It really surprised me that merely adding a class almost doubles the performance. 
If i want to maximize the performance, i just add a redundant class and train it along with the two classes (which is what i really need) ?

lastly, Will i be able to achieve 25+ fps on a Nvidia AGX Xavier ? i have never tried any of nvidia's single board solutions before.

Thanks for your great work on this repo.
